src/Symfony/Component/HttpKernel/DataCollector/ConfigDataCollector.php

```diff
@@ -67,11 +67,11 @@ public function collect(Request $request, Response $response, \Exception $except
             'env' => isset($this->kernel) ? $this->kernel->getEnvironment() : 'n/a',
             'debug' => isset($this->kernel) ? $this->kernel->isDebug() : 'n/a',
             'php_version' => PHP_VERSION,
+            'php_architecture' => PHP_INT_SIZE * 8,
+            'php_locale' => \Locale::getDefault() ?: 'n/a',
```

Comment:
Thanks for noticing this. What should we do here?

1. Relabel this to `Intl locale`
2. Get the PHP locale in a different way
3. Remove this for not being reliable